c/CMakeLists.txt

```diff
@@ -1 +1,2 @@
 add_subdirectory(parallel)
+add_subdirectory(experimental/stf/)
```

Comment:
@caugonnet Add two new cmake options here: `CCCL_C_ENABLE_Parallel` and `CCCL_C_ENABLE_Experimental` that both default to on. Update the presets to just switch between those.

Leave all of the subproject flags (tests, header tests, examples, etc) on by default as well, these can be omitted from the presets if so.

Side note: We should also be using consistent case throughout these options (using the name given to `project(...)` as the source-of-truth), but we're kinda sloppy about that already.